cloudomate/hoster/vps/linevast.py

```diff
@@ -79,6 +91,35 @@ def purchase(self, wallet, option):
         self._browser.submit_selected()
         self.pay(wallet, self.get_gateway(), self._browser.get_url())
 
+    def set_root_password(self, password):
```

Comment:
This method was deliberately removed in favor of a different, unified root password change mechanism.'
Please do not readd it.